mappings/net/minecraft/client/render/debug/DebugRenderer.mapping

```diff
@@ -17,6 +19,75 @@ CLASS net/minecraft/class_863 net/minecraft/client/render/debug/DebugRenderer
 	FIELD field_4537 worldGenAttemptDebugRenderer Lnet/minecraft/class_863$class_864;
 	FIELD field_4538 heightmapDebugRenderer Lnet/minecraft/class_863$class_864;
 	FIELD field_4539 structureDebugRenderer Lnet/minecraft/class_870;
+	METHOD <init> (Lnet/minecraft/class_310;)V
+		ARG 1 client
+	METHOD method_19429 drawText (Ljava/lang/String;DDDIF)V
+		ARG 0 text
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+		ARG 7 colorHex
+		ARG 8 scale
+	METHOD method_19692 drawBox (DDDDDDFFFF)V
+		ARG 0 minX
+		ARG 2 minY
+		ARG 4 minZ
+		ARG 6 maxX
+		ARG 8 maxY
+		ARG 10 maxZ
+		ARG 12 red
+		ARG 13 green
+		ARG 14 blue
+		ARG 15 alpha
+	METHOD method_19693 (Lnet/minecraft/class_1297;)Z
+		ARG 0 e
+	METHOD method_19694 rayTraceEntities (Lnet/minecraft/class_1297;I)Ljava/util/Optional;
+		ARG 0 observingEntity
+		ARG 1 distance
+	METHOD method_19695 drawBox (Lnet/minecraft/class_238;FFFF)V
+		ARG 0 box
+		ARG 1 red
+		ARG 2 green
+		ARG 3 blue
+		ARG 4 alpha
+	METHOD method_19696 drawBox (Lnet/minecraft/class_2338;FFFFF)V
+		ARG 0 position
+		ARG 1 scale
+		ARG 2 red
+		ARG 3 green
+		ARG 4 blue
+		ARG 5 alpha
+	METHOD method_19697 drawBox (Lnet/minecraft/class_2338;Lnet/minecraft/class_2338;FFFF)V
+		ARG 0 from
+		ARG 1 to
+		ARG 2 red
+		ARG 3 green
+		ARG 4 blue
+		ARG 5 alpha
+	METHOD method_20413 prepare ()V
 	METHOD method_3709 renderDebuggers (J)V
+		ARG 1 endTime
 	METHOD method_3710 shouldRender ()Z
+	METHOD method_3711 drawText (Ljava/lang/String;IIII)V
+		ARG 0 text
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 colorHex
```

Comment:
I want to encourage the use of `colorHex` because you typically are going to have to write colours in hexidecimal notation. 0xFFFFFFFF is easier to understand than -7778979 (or something like that). It's also a parallel to how colours are typically written in other places: #FFFFFF (but it's a number)

Reply:
The parameter name should correctly describe what it is. The method just wants an int representing a 24-bit color. How you decide to format it in your source code has nothing to do with what the parameter is.

Reply:
`colorHex` is clearer and easier to understand that the number is a colour _in hexidecimal notation_, better than just calling it `colorInt` or `tint`.